packages/studio/src/components/ActiveComponentPropEditors.tsx

```diff
@@ -32,13 +32,11 @@ export default function ActiveComponentPropEditors(props: {
 
   const propValues = activeComponentState.props;
   return (
-    <div className="overflow-x-auto">
```

Comment:
was it necessary for getting the correct dropdown styling to move the overflow into the EditorSidebar? I preferred having the scrolling limited to ActiveComponentPropEditors because I think the spacing for the top toggle bar becomes a little weird if it's scrollable (see video)

https://github.com/yext/studio/assets/88398086/e0f99782-5922-45c7-8edc-35c2b5ecff1a

Reply:
Fixed